starters/next-wordpress-starter/components/post.jsx

```diff
@@ -1,9 +1,13 @@
 import Image from "next/image";
 import Link from "next/link";
+import { getUrlPath } from "../lib/getUrlPath";
+import { IMAGE_URL } from "../lib/constants";
 
 export default function Post({
   post: { title, date, featuredImage, content },
 }) {
+  let srcUrl = getUrlPath(featuredImage.node.sourceUrl);
```

Comment:
```suggestion
  const srcUrl = featuredImage?.node?.sourceUrl && getUrlPath(featuredImage.node.sourceUrl);
```
Here and in `page.jsx`, we can add some optional chaining and short circuit evaluation so that we only pass in `featuredImage.node.sourceUrl` if it is truthy.